.github/workflows/ci.yml

```diff
@@ -52,4 +53,4 @@ jobs:
         if: |
           github.repository == 'denoland/deno_ast' &&
           github.ref == 'refs/heads/main'
-        run: deno run -A --no-check https://raw.githubusercontent.com/denoland/automation/0.2.0/tasks/release_on_crate_version_change.ts --publish
+        run: deno run -A --no-check https://raw.githubusercontent.com/denoland/automation/0.7.0/tasks/release_on_crate_version_change.ts --publish

```

Comment:
Use a task for this?

Reply:
I thought about it, but it's something that would only occur on the CI so I left it here.